ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yml

```diff
@@ -25,19 +20,24 @@ spec:
       type: string
     - name: asset_type
       type: string
-    - name: gpg_decryption_key_secret_name
-      description: The name of the Kubernetes Secret that contains the gpg decryption key.
-    - name: gpg_decryption_private_key_secret_key
-      description: The key within the Kubernetes Secret that contains the gpg private key.
-    - name: gpg_decryption_public_key_secret_key
-      description: The key within the Kubernetes Secret that contains the gpg public key.
-    - name: prow_kubeconfig_secret_name
-      description: The name of the Kubernetes Secret that contains the prow kubeconfig.
-    - name: prow_kubeconfig_secret_key
-      description: The key within the Kubernetes Secret that contains the prow kubeconfig.
-    - name: skip
-      description: "Set to 'true' to skip"
-      default: 'false'
+    - description: The name of the Kubernetes Secret that contains the gpg decryption key.
+      name: gpg_decryption_key_secret_name
+      type: string
+    - description: The key within the Kubernetes Secret that contains the gpg private key.
+      name: gpg_decryption_private_key_secret_key
+      type: string
+    - description: The key within the Kubernetes Secret that contains the gpg public key.
+      name: gpg_decryption_public_key_secret_key
+      type: string
+    - description: The name of the Kubernetes Secret that contains the prow kubeconfig.
+      name: prow_kubeconfig_secret_name
+      type: string
+    - description: The key within the Kubernetes Secret that contains the prow kubeconfig.
+      name: prow_kubeconfig_secret_key
+      type: string
+    - default: 'false'
+      description: Set to 'true' to skip
+      name: skip
```

Comment:
Aside from adding the type, it looks like this just changed order. Possible to have you keep the order to reduce the diff?